float32 X
float32 Y
float32 NewYaw
float32 DroneYaw
float32 YawVel
float32 PitchVel
float32 NewX
float32 LastX
float32 CurrentX
float32 NewY
float32 LastY
float32 CurrentY
int32 mode



